.github/workflows/ci.yaml

```
@@ -178,59 +186,63 @@ jobs:
           path: venv
           key: >-
             ${{ runner.os }}-${{ steps.python.outputs.python-version }}-${{
-            steps.generate-python-key.outputs.key }}
-          # Temporary disabling the restore of environments when bumping
-          # a dependency. It seems that we are experiencing issues with
-          # restoring environments in GitHub Actions, although unclear why.
-          # First attempt: https://github.com/home-assistant/core/pull/62383
-          #
-          # restore-keys: |
-          #   ${{ runner.os }}-${{ steps.python.outputs.python-version }}-base-venv-${{ env.CACHE_VERSION }}-${{ hashFiles('requirements.txt') }}-${{ hashFiles('requirements_test.txt') }}-
-          #   ${{ runner.os }}-${{ steps.python.outputs.python-version }}-base-venv-${{ env.CACHE_VERSION }}-${{ hashFiles('requirements.txt') }}-
-          #   ${{ runner.os }}-${{ steps.python.outputs.python-version }}-base-venv-${{ env.CACHE_VERSION }}-
+            needs.info.outputs.python_cache_key }}
       - name: Restore pip wheel cache
         if: steps.cache-venv.outputs.cache-hit != 'true'
         uses: actions/cache@v3.0.4
         with:
           path: ${{ env.PIP_CACHE }}
           key: >-
             ${{ runner.os }}-${{ steps.python.outputs.python-version }}-${{
-            steps.generate-pip-key.outputs.key }}
+            needs.info.outputs.python_cache_key }}
           restore-keys: |
             ${{ runner.os }}-${{ steps.python.outputs.python-version }}-base-pip-${{ env.PIP_CACHE_VERSION }}-${{ env.HA_SHORT_VERSION }}-
```

```suggestion
            ${{ runner.os }}-${{ steps.python.outputs.python-version }}-pip-${{ env.PIP_CACHE_VERSION }}-${{ env.HA_SHORT_VERSION }}-
```
You did remove `base` from the `generate-pip-key` step output above.